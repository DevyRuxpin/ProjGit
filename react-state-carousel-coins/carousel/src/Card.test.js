import { render } from "@testing-library/react";
import Card from "./Card";
import TEST_IMAGES from "./_testCommon.js";

// Smoke test
it("renders without crashing", function() {
  render(<Card 
    caption={TEST_IMAGES[0].caption}
    src={TEST_IMAGES[0].src}
    currNum={1}
    totalNum={3}
  />);
});

// Snapshot test
it("matches snapshot", function() {
  const { container } = render(<Card 
    caption={TEST_IMAGES[0].caption}
    src={TEST_IMAGES[0].src}
    currNum={1}
    totalNum={3}
  />);
  expect(container).toMatchSnapshot();
}); 