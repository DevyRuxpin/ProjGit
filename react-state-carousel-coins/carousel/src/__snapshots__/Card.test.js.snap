// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      testing image 1
    </h4>
    <img
      alt="testing image 1"
      class="Card-image"
      src="test1.com"
    />
    <small
      class="Card-small"
    >
      Image 
      1
       of 
      3
      .
    </small>
  </div>
</div>
`;
